zcbor/zcbor.py

```diff
@@ -1887,7 +1887,7 @@ def repeated_type_name(self):
         if self.self_repeated_multi_var_condition():
             name = self.raw_type_name()
             if self.val_type_name() == name:
-                name = name + "_"
+                name = name + "_r"
```

Comment:
What does `_r` indicate? Should we add a comment?

Reply:
Repeated part of the type. I added more comments to the functions.